snow/engine/snowman/transitive_test.go

```diff
@@ -30,298 +31,289 @@ var (
+			BytesV:  utils.RandomBytes(32),
+		}
+		chain[i] = root
+	}
+	return chain
+}
+
+func setup(t *testing.T) (ids.NodeID, validators.Manager, *common.SenderTest, *block.TestVM, *Transitive) {
 	require := require.New(t)
 
-	vals := validators.NewManager()
-	engCfg.Validators = vals
+	config := DefaultConfig(t)
 
 	vdr := ids.GenerateTestNodeID()
-	require.NoError(vals.AddStaker(engCfg.Ctx.SubnetID, vdr, nil, ids.Empty, 1))
-	require.NoError(engCfg.ConnectedValidators.Connected(context.Background(), vdr, version.CurrentApp))
-
-	vals.RegisterSetCallbackListener(engCfg.Ctx.SubnetID, engCfg.ConnectedValidators)
+	require.NoError(config.Validators.AddStaker(config.Ctx.SubnetID, vdr, nil, ids.Empty, 1))
+	require.NoError(config.ConnectedValidators.Connected(context.Background(), vdr, version.CurrentApp))
+	config.Validators.RegisterSetCallbackListener(config.Ctx.SubnetID, config.ConnectedValidators)
 
 	sender := &common.SenderTest{T: t}
-	engCfg.Sender = sender
+	config.Sender = sender
 	sender.Default(true)
 
 	vm := &block.TestVM{}
 	vm.T = t
-	engCfg.VM = vm
+	config.VM = vm
 
 	snowGetHandler, err := getter.New(
 		vm,
 		sender,
-		engCfg.Ctx.Log,
+		config.Ctx.Log,
 		time.Second,
 		2000,
-		engCfg.Ctx.Registerer,
+		config.Ctx.Registerer,
 	)
 	require.NoError(err)
-	engCfg.AllGetsServer = snowGetHandler
+	config.AllGetsServer = snowGetHandler
 
 	vm.Default(true)
 	vm.CantSetState = false
 	vm.CantSetPreference = false
 
-	gBlk := &snowman.TestBlock{TestDecidable: choices.TestDecidable{
-		IDV:     Genesis,
-		StatusV: choices.Accepted,
-	}}
-
 	vm.LastAcceptedF = func(context.Context) (ids.ID, error) {
-		return gBlk.ID(), nil
+		return GenesisID, nil
 	}
 
 	vm.GetBlockF = func(_ context.Context, blkID ids.ID) (snowman.Block, error) {
 		switch blkID {
-		case gBlk.ID():
-			return gBlk, nil
+		case GenesisID:
+			return Genesis, nil
 		default:
 			return nil, errUnknownBlock
 		}
 	}
 
-	te, err := New(engCfg)
+	te, err := New(config)
 	require.NoError(err)
 
 	require.NoError(te.Start(context.Background(), 0))
 
 	vm.GetBlockF = nil
 	vm.LastAcceptedF = nil
-	return vdr, vals, sender, vm, te, gBlk
-}
-
-func setupDefaultConfig(t *testing.T) (ids.NodeID, validators.Manager, *common.SenderTest, *block.TestVM, *Transitive, snowman.Block) {
-	engCfg := DefaultConfig(t)
-	return setup(t, engCfg)
+	return vdr, config.Validators, sender, vm, te
 }
 
 func TestEngineShutdown(t *testing.T) {
 	require := require.New(t)
 
-	_, _, _, vm, transitive, _ := setupDefaultConfig(t)
-	vmShutdownCalled := false
+	_, _, _, vm, engine := setup(t)
+
+	var vmShutdownCalled bool
 	vm.ShutdownF = func(context.Context) error {
 		vmShutdownCalled = true
 		return nil
 	}
-	vm.CantShutdown = false
-	require.NoError(transitive.Shutdown(context.Background()))
+
+	require.NoError(engine.Shutdown(context.Background()))
 	require.True(vmShutdownCalled)
 }
 
-func TestEngineAdd(t *testing.T) {
+func TestEngineDropsAttemptToIssueBlockAfterFailedRequest(t *testing.T) {
```

Comment:
Why is this change part of this PR?

Reply:
This just changed the name of the test, I felt it was more useful of a name then `TestEngineAdd`